objects/src/assets/nonfungible.rs

```diff
@@ -69,6 +72,11 @@ impl NonFungibleAsset {
         self.0
     }
 
+    /// Return ID of the faucet which issued this asset.
+    pub fn faucet_id(&self) -> AccountId {
+        AccountId::new_unchecked(self.0[FAUCET_ID_POS])
+    }
```

Comment:
Let me know if this should be changed to a checked version instead.

Reply:
I think this is fine as is.